examples/node-app/.babelrc

```diff
@@ -0,0 +1,33 @@
+{
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react"
+  ],
+  "plugins": [
+    ["@babel/plugin-proposal-decorators", { "legacy": true }],
```

Comment:
do we need this here if the module is loading compiled kepler.gl code?

Reply:
I copied the babel config file from other examples but you were right. Those two options are not necessary